### PR TITLE
Refacto various little thing

### PR DESCRIPTION
**Pull request by LaRiffle**
## Description
These are remaining few fixes and updates

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: syft/serde/compression.py [hunk 1]
```diff
@@ -75,6 +76,13 @@ def apply_no_compression(decompressed_input_bin) -> tuple:
     return decompressed_input_bin, NO_COMPRESSION
 
 
+scheme_to_compression = {
```
Comment (gmuraru) [inline syft/serde/compression.py:79]: Q: Could we move those default values as the first thing in the file (before function definitions)

Comment (LaRiffle) [reply syft/serde/compression.py:79]: @gmuraru  I wish I could but I need the functions to be defined first before I add them to the dict

File: syft/serde/compression.py [hunk 2]
```diff
@@ -75,6 +76,13 @@ def apply_no_compression(decompressed_input_bin) -> tuple:
...
+    NO_COMPRESSION: apply_no_compression,
+    LZ4: apply_lz4_compression,
```
[the remark this answers is not in the document]
Comment (LaRiffle) [reply syft/serde/compression.py:81]: @gmuraru done!

File: syft/serde/compression.py (hunk 1 above)
Comment (gmuraru) [reply syft/serde/compression.py:79]: Yeah :(

**Pull request merged**